README.md

```diff
@@ -7,56 +7,33 @@ Example Streem SDK project for IOS
 * Cocoapods 1.6 or later
 * ARKit compatible device with IOS 12 or later
 
-## Streem Functions
-
-| Core Functionality                                | Streem App        | SDK Status        |
-| ------------------------------------------------- | ----------------- | ----------------- |
-| Non-AR remote streeming                           | ✅ 				| ✅                |
-| Non-AR onsite streeming                           | ✅ 				| ✅                |
-| Pro-to-Pro calling                                | ❌ 				| ✅                |
-| Pro CallKit integration                           | ✅ 				| ✅                |
-| Customer CallKit integration                      | ❌ 				| ✅                |
-| Laser tool                                        | ✅ 				| ✅                |
-| Landscape Support                                 | ❌ 				| ✅                |
-| iPad Support                                      | ✅ 				| ✅                |
-| API Access (GET /streems)                         | ❌ 				| ✅                |
-| AR Remote Streem                                  | ✅ 				| ✅                |
-| AR Onsite Streem                                  | ✅ 				| ✅                |
-| AR Arrow Tool                                     | ✅ 				| ✅                |
-| Remote Streemshot                                 | ✅ 				| ✅                |
-| Onsite Streemshot                           	    | ✅ 				| ✅                |
-| Streemshot Editing                                | ✅					| ✅                    |
-| Streemshot Processing                             | ✅					|                   |
-| Onsite Recording                           	    | ✅ 				| ✅                |
-| Remote Recording                                  | ✅ 				|                   |
-
-
 ### Company/App Setup
 
 * Obtain your `company_id` from Streem
 * Provide your IOS bundle id for any apps you are going to use the Streem SDK in (later you will be able to do this from a self-service portal)
 * Streem will provide you with an `appId` and `appSecret` for each of your IOS apps
 * Now that you have your App IDs, follow the steps in the [CallKit Setup Instructions](docs/callkit.md)
+* The StreemKit SDK requires a navigation controller for streems. Please make sure your entry point to the StreemKit SDK is in a `NavigationController`
 
 ### Installation
 
 Currently Streem supports Cocoapods installation (Carthage, Swift Package Manager, and Manual to come later)
 
-Add a `source` to your `Podfile`:
+Have Streem add you to their cocoapods repository, and then add a `source` to your `Podfile`:
```

Comment:
Ty had said he wanted to change the repo to private and just give access to vetted partners. Is that not what we want to do?

Reply:
We can see what it would take, but cocoapods may not work well with a private repo.  Instead, we can limit the ability to use by who gets App ID/Secrets.  If Ty wants to restrict that, then we should add a feature flag for whether that shows up in admin.

Reply:
I did some research, it looks straight forward enough: https://guides.cocoapods.org/making/private-cocoapods.html. The next issue I'm working on I was going to test it out. But if we'd rather not make the repo private we can certainly go the App ID/Secrets route.